### PR TITLE
Upgrade PF and get rid of custom CSS on settings page

### LINKED ISSUE #950: [Deps]: Bump version of PatternFly to 2023.01 release
### Feature description

"@patternfly/patternfly": "4.222.4" -> "4.224.2",
"@patternfly/react-catalog-view-extension":  "4.93.15" -> "4.95.1",
"@patternfly/react-charts": "6.94.15" -> "6.94.18"
"@patternfly/react-core": "4.267.6" -> "4.276.6",
"@patternfly/react-icons": "4.93.3" -> "4.93.6",
"@patternfly/react-styles": "4.92.3" -> "4.92.6",
"@patternfly/react-table": "4.112.6" -> "4.112.39",
"@patternfly/react-virtualized-extension": "4.88.82" -> "4.88.113",

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
## Description

Closes #950 
Because PF changed the `onChange` prop order for `SearchInput` to match the DOM event (event as the first parameter and value as the second), we also need to make some small updates to match that change.
Also, update the user management page and cluster settings page, get rid of the Form component, and use the plain bordered Card component instead to get rid of the custom CSS styles.

## How Has This Been Tested?

The visual styles are slightly changed on the settings page, so go to the settings page to manually see if anything looks weird. The functionalities keep the same.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by andrewballantyne** [comment]
Holding to prevent merging in this release -- this is for next release.

**Comment by DaoDaoNoCode** [comment]
Verified #977 is fixed by bumping the PF version.

Screenshot 2023-02-23 at 1 57 41 PM

**Review by andrewballantyne — APPROVED** [review]

**Comment by andrewballantyne** [comment]
Daaaang, those settings pages look sooooo much better. Love the head boldness, the clean borders and spacing.

Screenshot 2023-03-01 at 6 20 55 PM
Screenshot 2023-03-01 at 6 21 16 PM

File: frontend/src/pages/clusterSettings/ClusterSettings.tsx [hunk 1]
```diff
@@ -426,22 +407,74 @@ const ClusterSettings: React.FC = () => {
...
+                  />
+                </StackItem>
+              </Stack>
+            </SettingSection>
+          </StackItem>
+        )}
+        <StackItem>
+          <Button
+            data-id="submit-cluster-settings"
+            isDisabled={saving || !isSettingsChanged || !!notebookTolerationSettings.error}
```
Comment (lucferbux) [inline frontend/src/pages/clusterSettings/ClusterSettings.tsx:469]: Button should be disable if `pvcSize` is empty.

File: frontend/src/pages/clusterSettings/ClusterSettings.tsx [hunk 2]
```diff
@@ -175,242 +170,228 @@ const ClusterSettings: React.FC = () => {
...
+                                : newValue;
+                            // if the hour is max, then the minute can only be set to 0
+                            if (newValue === MAX_HOUR && minute !== MIN_MINUTE) {
+                              setMinute(MIN_MINUTE);
+                            }
+                            setHour(newValue);
+                          }}
+                        />
+                        <InputGroupText variant={InputGroupTextVariant.plain}>hours</InputGroupText>
+                        <TextInput
```
Comment (lucferbux) [inline frontend/src/pages/clusterSettings/ClusterSettings.tsx:310]: Reeeally weird issue in safari, tested in firefox and chrome and works great.

Screenshot 2023-03-02 at 11 17 16

**Review by lucferbux — CHANGES_REQUESTED** [review]

File: frontend/src/pages/clusterSettings/ClusterSettings.tsx (hunk 2 above)
Comment (DaoDaoNoCode) [reply frontend/src/pages/clusterSettings/ClusterSettings.tsx:310]: Hmmm, weird. But I found put `Radio` inside `FlexItem` could solve this problem, nice catch!

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**